data/EddieHubCommunity/testimonials/krishstwt.json

```
@@ -1,5 +1,5 @@
-{
-  "title": "Best OSS Community 🔥",
-  "description": "EddieHub community is amazing! This is where my journey in the world of coding and development began, and I am so grateful to have found such an incredible community. The community at EddieHub truly goes above and beyond to help everyone in every possible way they can. I had no experience when I first joined but with the help and guidance of the community, I am now able to help others and continue learning new skills. I love the supportive atmosphere and friendly community members. Eddie, the moderators, and every member of this community are truly awesome. I cannot recommend EddieHub enough!",
-  "date": "2023-01-10"
-}
+{
+  "title": "Best OSS Community 🔥",
+  "description": "EddieHub community is amazing! This is where my journey in the world of coding and development began, and I am so grateful to have found such an incredible community. The community at EddieHub truly goes above and beyond to help everyone in every possible way they can. I had no experience when I first joined but with the help and guidance of the community, I am now able to help others and continue learning new skills. I love the supportive atmosphere and friendly community members. Eddie, the moderators, and every member of this community are truly awesome. I cannot recommend EddieHub enough!",

```

[praise] I agree @krishstwt! :) Eddiehub is the greatest OSS community ever! :) Without you all, I would've never accomplished a lot of things I mentioned on Twitter! :)   
```suggestion
  "description": "EddieHub community is amazing! This is where my journey in the world of coding and development began, and I am so grateful to have found such an incredible community. The community at EddieHub truly goes above and beyond to help everyone in every possible way they can. I had no experience when I first joined but with the help and guidance of the community, I am now able to help others and continue learning new skills. I love the supportive atmosphere and friendly community members. Eddie, the moderators, and every member of this community are truly awesome. I cannot recommend EddieHub enough!",

```

---

I think all of agree <3